### ISSUE TITLE
The pages that appear when creating a group are not responsive

### ISSUE THREAD

**Issue by vplasencia** [bandada-infra/bandada#559]
## Description

The pages that appear when creating a group are not responsive, it would be nice to make them responsive. 

<img width="297" alt="Screenshot 2024-09-22 at 12 01 02 PM" src="https://github.com/user-attachments/assets/c2b69b78-caca-4a81-8be3-e983fb7fa754">

**Comment by Benjtalkshow** [comment]
I am applying to this issue via [OnlyDust platform](https://app.onlydust.com/p/bandada).

### My background and how it can be leveraged
Hello @vplasencia,
Can you assign this task to me, i am fully ready to deliver on time. Thnaks

**Comment by vplasencia** [comment]
Hey @Benjtalkshow do you want me to assign this issue to you?

**Comment by Benjtalkshow** [comment]
> Hey @Benjtalkshow do you want me to assign this issue to you?

Yes. I want to fix the issue. Assign the issue to me. i can fix it ASAP.

**Comment by Benjtalkshow** [comment]
> > Hey @Benjtalkshow do you want me to assign this issue to you?
> 
> Yes. I want to fix the issue. Assign the issue to me. i can fix it ASAP.

Thanks

**Comment by yagopajarino** [comment]
Hi @vplasencia found this issue related to #557 and #558 could you also assign this one to me?
Thanks! 🚀

**Comment by vplasencia** [comment]
Hey @yagopajarino! Sure, done. Let us know if you have any questions.
